composer.json

```
@@ -90,7 +90,7 @@
     "symfony/polyfill-mbstring": "^1.31.0",
     "symfony/web-link": "^6.4.13",
     "symfony/yaml": "^6.4.18",
-    "typo3/phar-stream-wrapper": "^4.0.0",
+    "typo3/phar-stream-wrapper": "^v3.1.8",
```

```suggestion
    "typo3/phar-stream-wrapper": "^3.1.8",
```
Shouldn't that be without the "v"? We don't use it elsewhere in that section of that file.

---

Afaik it doesn't really matter.

---

you should remove the v - semver specifically says not to use the v